handlers/routeservice.go

```diff
@@ -62,6 +62,33 @@ func (r *RouteService) ServeHTTP(rw http.ResponseWriter, req *http.Request, next
 	routeServiceURL := reqInfo.RoutePool.RouteServiceUrl()
 	if routeServiceURL == "" {
 		// No route service is associated with this request
+		if r.config.StrictSignatureValidation() && len(req.Header.Values(routeservice.HeaderKeySignature)) > 0 {
+			// Someone is still setting the header, possibly trying to impersonate a route service
+			// request. We will validate the request to ensure the header is valid.
+			// TODO: what do we have to provide as app URL?
```

Comment:
Should this TODO still be here?

Reply:
Nope :smile:

Reply:
That field is never read and we should probably remove it, but the TODO should not be in this commit, I'll remove it.